CHANGELOG.md

```diff
@@ -1,3 +1,159 @@
+# 65.0.0
+
+TextFields placehold text layout adjustment.
+Bottom App Bar got a `floatingButtonVerticalOffset` property.
+ActionSheets continued work. (still in alpha).
+AlertController in dialogs: Incremental improvements to the to bring it up to the design guidelines.
```

Comment:
"Incremental improvements to the" what?

Reply:
thanks for catching me here.
haha sorry about that. I had a sentence the reordered it without fully proofreading it.